rest_api/javascript/click_to_queue.html

```diff
@@ -65,7 +65,6 @@
       // Find queue ID by matching the queue name to button queue id
       var queue = queues
         .filter(function(queue) { return queue.id === buttonQueueId; })
-        .map(function(queue) { return queue })[0];
```

Comment:
The `map` was unnecessary, but pretty sure the removal of `[0]` changes the behavior.